.github/workflows/main.yml

```diff
@@ -0,0 +1,33 @@
+name: CI
+on: [push, pull_request]
+
+jobs:
+  test:
+    name: Test
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@master
+    - name: Update rustup
+      run: rustup self update
+    - name: Install Rust
+      run: |
+        rustup set profile minimal
+        rustup install nightly
```

Comment:
```suggestion
        rustup install nightly -c rust-docs
```

Reply:
I get `error: Found argument '-c' which wasn't expected, or isn't valid in this context` ?

Reply:
My memory failed me. This could mark as resolved.

Reply:
Oh, it is available with `toolchain install`, not the `install` alias.  That's a little confusing.